src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/ValidationService.kt

```diff
@@ -575,14 +576,22 @@ class ValidationService(
 
   private fun validateDtoIsNotRecall(prisonApiSourceData: PrisonApiSourceData): List<ValidationMessage> {
     val validationMessages = mutableListOf<ValidationMessage>()
+    var bookingHasDto = false
+    var bookingHasRecall = false
     prisonApiSourceData.sentenceAndOffences.forEach() {
 
       val sentenceCalculationType = SentenceCalculationType.from(it.sentenceCalculationType)
-      val hasRecall = it.terms.any { terms ->
+      val hasDtoRecall = it.terms.any { terms ->
         terms.code == SentenceTerms.BREACH_OF_SUPERVISION_REQUIREMENTS_TERM_CODE || terms.code == SentenceTerms.BREACH_DUE_TO_IMPRISONABLE_OFFENCE_TERM_CODE
       }
       val hasDto = sentenceCalculationType == DTO || sentenceCalculationType == DTO_ORA
-      if (featureToggles.dto && hasDto && hasRecall) {
+      if (hasDto) {
```

Comment:
may be misreading bot looks like hasDto and boookingHasDto is the same thing? in which case cant we just use one ? e.g. drop bookingHasDto